sdk/webpubsub/Azure.Messaging.WebPubSub/README.md

```diff
@@ -80,7 +80,7 @@ serviceClient.SendToAll("Hello World!");
 ```C# Snippet:WebPubSubSendJson
 var serviceClient = new WebPubSubServiceClient(new Uri(endpoint), "some_hub", new AzureKeyCredential(key));
 
-serviceClient.SendToAll(
+serviceClient.SendToAll("application/json",
```

Comment:
Consider either updating https://github.com/Azure/azure-webpubsub/blob/main/docs/references/server-sdks/csharp-server-sdks.md as well or working with them to link to this file so we don't have out of date copies.